examples/example_vendor_slc_extension/projects/example-vendor-soc-with-buttons.slcp

```diff
@@ -36,7 +36,7 @@ quality: production
 component:
   - id: ot_platform_abstraction_core
   - id: ot_psa_crypto
-  - id: ot_thirdparty
+  - id: ot_mbedtls
   - id: uartdrv_usart
```

Comment:
Why doesn't this slcp include the following which was added to the other slcps.

 - id: ot_stack_features_config
    from: ot-efr32

Reply:
Good catch!